ckanext/restricted/logic.py

```diff
@@ -98,16 +100,17 @@ def restricted_check_user_resource_access(user, resource_dict, package_dict, use
         return {'success': True}
 
     # Get organization list
-    user_organization_dict = {}
+    user_organization_dict = getattr(user_obj, 'org_dict', {})
```

Comment:
I think it's possible that here `user_obj` in `None`

Reply:
That's fine isn't it? That's in part the purpose of getattr right? i.e...
```
>>> getattr(None, 'org_dict', {})
{}
```